teamsyncd/teamsync.cpp

```diff
@@ -36,9 +36,18 @@ TeamSync::TeamSync(DBConnector *db, DBConnector *stateDb, Select *select) :
         m_start_time = steady_clock::now();
         auto warmRestartIval = WarmStart::getWarmStartTimer(TEAMSYNCD_APP_NAME, "teamd");
         m_pending_timeout = warmRestartIval ? warmRestartIval : DEFAULT_WR_PENDING_TIMEOUT;
-        m_lagTable.create_temp_view();
-        m_lagMemberTable.create_temp_view();
-        WarmStart::setWarmStartState(TEAMSYNCD_APP_NAME, WarmStart::INITIALIZED);
+        if (m_pending_timeout != MAXIMUM_WARMRESTART_TIMER_VALUE)
```

Comment:
I'd suggest rename MAXIMUM_WARMRESTART_TIMER_VALUE to NO_WARMRESTART_TIMER_VALUE or DISABLE_WARMRESTART_TIMER_VALUE

Reply:
> I'd suggest rename MAXIMUM_WARMRESTART_TIMER_VALUE to NO_WARMRESTART_TIMER_VALUE or DISABLE_WARMRESTART_TIMER_VALUE

UPDATED